CMakeLists.txt

```diff
@@ -137,14 +137,14 @@ if(LUAU_EXTERN_C)
     # enable extern "C" for VM (lua.h, lualib.h) and Compiler (luacode.h) to make Luau friendlier to use from non-C++ languages
     # note that we enable LUA_USE_LONGJMP=1 as well; otherwise functions like luaL_error will throw C++ exceptions, which can't be done from extern "C" functions
     target_compile_definitions(Luau.VM PUBLIC LUA_USE_LONGJMP=1)
```

Comment:
I was unsure about if this should also be made private.

There doesn't appear to be anything in the header that makes use of it and a consumer may want to know about this.